### PR TITLE
Feat/Accessibility phase 1

### PR DESCRIPTION
**Pull request by ArnonZ**
- Added accessibility for Avatar and Badge

### CODE REVIEW

File: src/components/avatar/index.js [hunk 1]
```diff
@@ -294,6 +294,7 @@ export default class Avatar extends PureBaseComponent {
 
   render() {
     const {
+      accessibilityLabel,
```
Comment (ethanshar) [inline src/components/avatar/index.js]: I think you forgot to remove it, it should also fail lint

File: src/components/badge/index.js [hunk 2]
```diff
@@ -97,6 +97,17 @@ export default class Badge extends PureBaseComponent {
     }
   }
 
+  getAccessibilityProps() {
+    const {onPress, icon, label} = this.props;
+    
+    return {
+    accessibilityLabel: icon ? 'badge' : `badge ${label}` || 'badge',
```
Comment (ethanshar) [inline src/components/badge/index.js]: what is this
```
`badge ${label}` || 'badge'
```
you're using pipe for a fallback value?

Comment (ethanshar) [reply src/components/badge/index.js]: also consider using `imagebutton` role for when the badge has onPress with an icon

Comment (ArnonZ) [reply src/components/badge/index.js]: Fixed.
Regarding `imagebutton`, it can have onPress in all badge variation (icon, text, no text - just bg color).

**Pull request merged**